geoengine/api.py

```diff
@@ -503,8 +503,8 @@ class OgrMetadata(MetaDataDefinition):  # pylint: disable=too-few-public-methods
 
 class DatasetProperties(TypedDict):  # pylint: disable=too-few-public-methods
     '''The properties of a dataset'''
-    id: Optional[str]
-    name: str
+    name: Optional[str]
```

Comment:
it's only optional for AddDatasetProperties right?

Reply:
[0d2d3dd](https://github.com/geo-engine/geoengine-python/pull/128/commits/0d2d3dd1e1ac45ac0860bad84087d4dc90b67fca)